doc/Language/regexes.pod6

```diff
@@ -61,21 +61,64 @@ and L<multi line/embedded comments|
 
     say '2015-12-25'.match($regex);     # OUTPUT: «｢2015-12-25｣␤»
 
-=head1 Literals
+=head1 Literals and metacharacters
 
-The simplest use case for a regex is a match against a string literal:
+A regex describes a pattern to be matched in terms of literals and
+metacharacters. Alphanumeric characters and the underscore C<_> constitute the
+literals: these characters match themselves and nothing else. Other characters
+are metacharacters and may, as such, have a special meaning, either alone (such
+as the dot C<.>, which acts as a wildcard) or together with other characters in
+larger metasyntactic constructs (such as C«<?before ...>», which defines a
+lookahead assertion). But before looking at the metacharacters and their
+particular uses, let's explore the relation between literals and metacharacters
+in some more detail.
+
+In its simplest form a regex comprises only literals:
 
     if 'properly' ~~ / perl / {
-        say "'properly' contains 'perl'";
+        say "'properly' contains 'perl'";   # OUTPUT: «'properly' contains 'perl'␤»
     }
 
```

Comment:
Maybe include here a paragraph explaining that quoting will match literally any string.